lib/OpenQA/WebAPI.pm

```diff
@@ -343,7 +343,8 @@ sub startup {
     $api_ro->post('/jobs/restart')->name('apiv1_restart_jobs')->to('job#restart');
 
     my $job_r = $api_ro->route('/jobs/:jobid', jobid => qr/\d+/);
-    $api_public_r->route('/jobs/:jobid', jobid => qr/\d+/)->get('/')->name('apiv1_job')->to('job#show');
+    $api_public_r->route('/jobs/:jobid', jobid => qr/\d+/)->name('apiv1_job')->to('job#show');
+    $api_public_r->route('/jobs/:jobid/details', jobid => qr/\d+/)->name('apiv1_job')->to('job#show', details => 1);
```

Comment:
Wasn't our idea to use the .json render route instead of an explicit API route? Or maybe even better to add the same to both?

Reply:
We already had the API route, I just added the details part. Adding support to render it at `/tests/:jobid/details` would have been only redundant when what we actually want is the pure json data.

Reply:
I'm not sure I get it but nevermind